validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/ValidatorLoader.java

```diff
@@ -49,6 +51,29 @@
     return validators;
   }
 
+  private static Map<BLSPublicKey, ValidatorInfo> createLocalSignerValidatorInfo(
+      final ArtemisConfiguration config) {
+    return loadValidatorKeys(config).stream()
+        .collect(
+            Collectors.toMap(
+                BLSKeyPair::getPublicKey,
+                blsKeyPair -> new ValidatorInfo(new LocalMessageSignerService(blsKeyPair))));
+  }
+
+  private static Map<BLSPublicKey, ValidatorInfo> createExternalSignerValidatorInfo(
+      final ArtemisConfiguration config) {
+    return config.getValidatorExternalSigningPublicKeys().stream()
+        .collect(
+            Collectors.toMap(
+                identity(),
+                publicKey ->
+                    new ValidatorInfo(
+                        new ExternalMessageSignerService(
+                            config.getValidatorExternalSigningUrl(),
```

Comment:
we only support one external signer?

Reply:
Yes at the moment. It wouldn't be too difficult to add support for multiple external signers though. The config would be become little more complex though. Do you think we should support multiple external signers?

Reply:
Thinking about this more, I don't think we need to support multiple external signers at the moment. And more than likely we will probably share the same external signer for several Teku instances. Certainly can quite easily add this in the future if it becomes apparent we do need this.